transcom/mymove
# B-19726-INT Add EMPLID to SC Customer create

## [Agility ticket](https://www13.v1host.com/USTRANSCOM38/assetdetail.v1?number=B-19726)

## Summary

Part of the effort to have USCG EMPLID accessible. Office users can optionally enter an EMPLID for USCG customers.

### How to test

1. Access the office app as a SC with safety move privileges
2. Create a new customer
3. Select Coast Guard as the branch and confirm the EMPLID appears
4. try different combinations of safety move/coast guard
5. try and break it

To test AC involving clearing the EMPLID for a safety move
1. Create a new customer/move
2. Login as an SC and locate the customer, and add a new move for that customer
3. Add a new safety move for that customer
4. Login as the customer and confirm your EMPLID has been cleared out
